cron/cron.go

```diff
@@ -82,18 +83,31 @@ func (c *Cron) AddJob(data JobData) error {
 		cmdPrefix,
 		cmdSuffix,
 	)
+	serviceName := data.Name
+	if data.ServiceName != "" {
+		serviceName = data.ServiceName
+	}
 	cmd := fmt.Sprintf(
-		`%s -l "com.df.cron=true" -l "com.df.cron.name=%s" -l "com.df.cron.schedule=%s" %s %s`,
+		`%s -l "com.df.cron=true" -l "com.df.cron.name=%s" -l "com.df.cron.schedule=%s" --name %s %s %s`,
 		cmdPrefix,
 		data.Name,
 		data.Schedule,
+		serviceName,
 		cmdLabel,
 		strings.Trim(cmdSuffix, " "),
 	)
+
+	_, err := exec.Command("/bin/sh", "-c", cmd).CombinedOutput()
```

Comment:
This will create the service immediately instead of scheduling it. What is a user wants to create a service that is executed every day at midnight? This would create it immediately and then schedule it to run every day at midnight. Instead, it should be that, following the same example, the service is created at the next midnight and scaled to 1 every midnight after that.

What do you think?